assets/styles/main.css

```diff
@@ -0,0 +1,129 @@
+@font-face {
+    font-family: 'Lucida Grande';
+    src: url(../fonts/lucida-grande/LucidaGrande.ttf);
+}
+
+@font-face {
+    font-family: 'Lucida Grande Bold';
+    src: url(../fonts/lucida-grande/LucidaGrandeBold.ttf);
+}
+
+body{
+    min-width: 900px;
+}
+
+
+.cover{ 
+    min-width: 900px;
+    height: 580px;
+    overflow: hidden;
+    background: url(../imgs/your_verse_hero.jpg) 50% 100% no-repeat;
+    
+}
+
+.cover .title,
+.cover .subtitle,
+.cover .foottitle {
+    display: block;
+    margin-left: auto;
+    margin-right: auto
+}
+
+header{
+    margin: 0 100px;
+}
+
+nav{
+    height: 125px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+}
+
+.navbar{
+    height: 36px;
+    width: 980px;
+    background-image: linear-gradient(rgb(107,107,107) 50%, rgb(122,122,122) 50%);
+    margin: 0 auto;
+    border-radius: 4px;
+    border-style: solid;
+    border-width: .2px;
+    border-color: rgb(80,80,80);
+    display: flex;
+    flex-direction: horizontal;
+    color: rgb(240,240,240);
+    font-family: 'Lucida Grande'
+}
+
+.cover ul li{
+    width: 102px;
+    text-align: center; 
+    line-height: 36px; 
+    border-style: solid;
+    border-width: .5px;
+    border-color: rgb(80,80,80); 
+}
+.cover ul li:hover{
+    background-image: linear-gradient(rgb(67,67,67) 50%, rgb(72,72,72) 50%);
+}
+.search{
+    width: 170px;
+}
+
+aside{
+    display: flex;
+}
+.promo{
+    width: 25%;
+    min-width: 300px;
+    height: 200px;
+    background-color: blue;
+    border-style: solid;
+    border-color: white;
+    border-width: 1px
+}
+.promo1{
+    background-image: url(../imgs/promo_iphone5s.jpg);
+    background-position: center;
+}
+.promo2{
+    background-image: url(../imgs/promo_iphone5c.jpg);
+    background-position: center;
+}
+.promo3{
+    background-image: url(../imgs/promo_30_years.jpg);
+    background-position: center;
+}
+.promo4{
+    background-image: url(../imgs/promo_supplier_responsibility.jpg);
+    background-position: center;
+}
+
+footer{
+    display: flex;
+    flex-wrap: wrap;
+    font-family: 'Lucida Grande';
+    color: rgb(110,110,110);
+}
+
+footer ul,
+footer div
+{
+    display: flex;
+    flex-direction: horizontal;
```

Comment:
Same as in line 53 `flex-direction` does not accept a `horizontal` as value

Reply:
Done